### PR TITLE
Docs landing page

### PR DESCRIPTION
**Pull request by mviitane**
**What this PR includes**

- Updated the docs landing page

Signed-off-by: mviitanen <mviitanen@mirantis.com>

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -1,26 +1,32 @@
-# Overview
+# k0s - The Zero Friction Kubernetes
 
 ![k0s logo](img/k0s-logo-full-color-light.svg)
 
-[k0s](https://k8slens.dev/kubernetes) is an all-inclusive Kubernetes distribution, which is configured with all of the features needed to build a Kubernetes cluster and packaged as a single binary for ease of use.
+k0s is an open source, all-inclusive Kubernetes distribution, which is configured with all of the features needed to build a Kubernetes cluster. Due to simple design, flexible deployment options and modest system requirements, k0s is well suited for
```
Comment (s0j) [inline docs/README.md:5]: "Due to its simple design"

File: docs/README.md [hunk 2]
```diff
@@ -1,26 +1,32 @@
...
 
-k0s fits well in any cloud environment, but can also be used in IoT gateways, Edge and Bare metal deployments due to its simple design, flexible deployment options and modest system requirements.
+- Any cloud
+- Bare metal
+- Edge and IoT
+
+k0s drastically reduces the complexity of installing and running a CNCF certified Kubernetes distribution. With k0s new clusters can be bootstrapped in minutes and developer friction is reduced to zero. This allows anyone with no special skills or expertise in Kubernetes to easily get started.
+
+k0s is distributed as a single binary with zero host OS dependencies besides the host OS kernel. It works with any Linux without additional software packages or configuration. Any security vulnerabilities or performance issues can be fixed directly in the k0s distribution that makes it exceedingly straightforward to keep the clusters up-to-date and secure.
```
Comment (s0j) [inline docs/README.md:13]: Either change 'exceedingly' to 'extremely' or just remove the word completely.

File: docs/README.md (hunk 1 above)
Comment (mviitane) [reply docs/README.md]: Thanks, updated.

File: docs/README.md (hunk 2 above)
Comment (mviitane) [reply docs/README.md]: Thanks, updated.

**Review by kke — APPROVED** [review]

**Pull request merged**